common/src/main/proto/TransportMessages.proto

```diff
@@ -209,17 +209,18 @@ message PbRequestSlotsResponse {
 message PbRevive {
   string applicationId = 1;
   int32 shuffleId = 2;
-  int32 mapId = 3;
-  int32 attemptId = 4;
-  int32 partitionId = 5;
-  int32 epoch = 6;
-  PbPartitionLocation oldPartition = 7;
-  int32 status = 8;
+  repeated int32 mapId = 3;
+  repeated int32 partitionId = 4;
+  repeated int32 epoch = 5;
+  repeated PbPartitionLocation oldPartition = 6;
+  repeated int32 status = 7;
 }
 
 message PbChangeLocationResponse {
-  int32 status = 1;
-  PbPartitionLocation location = 2;
+  repeated int32 endedMapId = 1;
+  repeated int32 partitionId = 2;
```

Comment:
Also can use struct message for partitionId/status/Location

Reply:
done